modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Build.tsx

```diff
@@ -29,13 +29,28 @@ import {
 	TestrayCaseResult,
 	testrayCaseResultImpl,
 } from '../../../../services/rest';
-import {searchUtil} from '../../../../util/search';
+import {SearchBuilder} from '../../../../util/search';
 import useBuildTestActions from './useBuildTestActions';
 
+interface OutletContext {
```

Comment:
We use `type` instead of `interface`

```suggestion
type OutletContext = {
```